storage/ndb/src/kernel/vm/mt.cpp

```diff
@@ -7627,34 +7708,13 @@ run_job_buffers(thr_data *selfptr,
   Uint32 signal_count = 0;
   Uint32 signal_count_since_last_zero_time_queue = 0;
 
+  signal_count = execute_high_prio_signals(sig,
```

Comment:
In the ldm thread, send_sum and flush_sum are now always the same in this patch.
In the previous implementation, mt_job_thread_main() would perform do_flush() in every iteration and only execute do_send() once in handle_scheduling_decisions when the selfptr->m_max_signals_before_send_flush threshold was reached. 
This patch changes the frequency of do_flush(), accumulating signals in the local buffer until calling handle_scheduling_decisions() to execute do_send().